frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md

```diff
@@ -742,10 +690,44 @@ The Dependent Field(s) will be displayed only when the current value of the Cont
   </tr>
 </table>
 
+
+#### 21. hidden
+
+**x-descriptors**
+
+This descriptor allows you to exclude non-user facing fields from the &quot;auto-generated creation form&quot; by specifying them as &quot;hidden&quot; schema fields. See example for Portworx Operator:
+
+```yaml
+…
+ - displayName: HIDDEN FIELDS - Network and all its children fields
+   description: >-
+      HIDDEN FIELDS - The network related fields to be used by Portworx for data traffic.
+   path: network
+   x-descriptors:
+     - 'urn:alm:descriptor:com.tectonic.ui:hidden'
+ - displayName: HIDDEN FIELDS - Start Port
+   description: >-
+      HIDDEN FIELDS - It is the starting port in the range of ports used by Portworx.
+   path: startPort
+   x-descriptors:
+     - 'urn:alm:descriptor:com.tectonic.ui:hidden'
+…
+```
+
+**Usage**
+1. You can apply &quot;hidden&quot; to any schema property path (doesn't have to be a &quot;leaf&quot; in the schema). All the children properties, if any, will be hidden as well.
+2. The descriptor would hide that property and all its children fields.
```

Comment:
This seems like it just repeats the same info in the first point. I would either remove the second sentence in the first point or just remove this point.
```suggestion
2. The descriptor will hide the target property and all of its child fields.
```